libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -296,7 +297,7 @@ - (void)revokeAuthentication {
 
 - (BOOL)handleIDPAuthenticationResponse:(NSURL *)appUrlResponse {
     self.authInfo = [[SFOAuthInfo alloc] initWithAuthType:SFOAuthTypeIDP];
-    [SFSDKAppFeatureMarkers registerAppFeature:kSFAppFeatureSafariBrowserForLogin];
+    [SFSDKAppFeatureMarkers registerAppFeature:kSFAppFeatureIDPLogin];
```

Comment:
I don't think we need to record this marker every time an app uses IDP to authenticate. We should probably move this inside `- (void)setIdpEnabled:(BOOL)idpEnabled`, where we add the marker if `YES` is passed in and remove the marker if `NO` is passed in.

Reply:
Also, the marker for `kSFAppFeatureSafariBrowserForLogin` needs to be added somewhere else where browser based login is enabled/disabled.

Reply:
@bhariharan goof point we can perhaps tie the Safari marker to the advanced auth enablement as well.